modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ElmClientCodegen.java

```diff
@@ -160,6 +165,10 @@ public ElmClientCodegen() {
     public void processOpts() {
         super.processOpts();
 
+        if (StringUtils.isEmpty(System.getenv("ELM_FORMAT_PATH"))) {
```

Comment:
Do we need a system variable for this? Can we not run `elm-format` from the PATH?

Reply:
I feel it makes more sense to use it if it is on the PATH, but if this is consistent with other generators I would keep it this way.